src/MahApps.Metro/Controls/Badged.cs

```diff
@@ -50,5 +66,30 @@ private void OnBadgeChanged(object sender, RoutedPropertyChangedEventArgs<object
                 }
             }
         }
+
+
+
+        /// <summary>
+        /// Gets or Sets the <see cref="DataTemplate"/> for the Badge
+        /// </summary>
+        public DataTemplate BadgeTemplate
+        {
+            get { return (DataTemplate)GetValue(BadgeTemplateProperty); }
+            set { SetValue(BadgeTemplateProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or Sets the <see cref="DataTemplateSelector"/> for the Badge
+        /// </summary>
+        public DataTemplateSelector BadgeTemplateSelector
+        {
+            get { return (DataTemplateSelector)GetValue(BadgeTemplateSelectorProperty); }
+            set { SetValue(BadgeTemplateSelectorProperty, value); }
+        }
+
+        
+
+
+
```

Comment:
@timunie This can be moved to https://github.com/ControlzEx/ControlzEx/blob/develop/src/ControlzEx/BadgedEx.cs

with this format

DependencyProperty 

Property

DependencyProperty 

Property